### DISCUSSION TITLE
Add third-party extensions

### DISCUSSION THREAD

**Discussion by mrGoonies** [zed-industries/zed#8354] [Feature Requests]
Hello everyone, this is the first time that I have been motivated to contribute to an open source project and in this case I would like to create extensions for this text editor, but my question is how can I add my extensions to the project (I understand that I should read the **CONTRIBUTING.md** file) and what advice could you give me as a newbie in this world. Everything is appreciated and thank you very much.

*PS: If this post does not correspond, I will delete it and apologize for the error.*

**Comment by Moshyfawn** [comment]
Zed currently supports language and theme extensions, which can be found in [the extensions repository](https://github.com/zed-industries/extensions). You can't yet extend the functionality of Zed using a plugin system.

Instructions on how to create extensions can be found in [the authoring extensions document](https://github.com/zed-industries/extensions/blob/main/AUTHORING_EXTENSIONS.md).

As for advice, if you're interested in contributing to Zed, be sure to check out [the current top ranking issues list](https://github.com/zed-industries/zed/issues/5393), as the items listed there are the ones that the Zed community is currently prioritising.

Also, be sure to join [Zed's Discord](https://discord.gg/k4pFw73j), where most of the community discusses all things Zed and more.
